src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -457,4 +475,28 @@ object Task {
       else None
     }
   }
+
+  implicit class TaskStatusComparison(val task: Task) extends AnyVal {
+    def isReserved: Boolean = compareTo(MarathonTaskStatus.Reserved)
+    def isCreated: Boolean = compareTo(MarathonTaskStatus.Created)
+    def isError: Boolean = compareTo(MarathonTaskStatus.Error)
+    def isFailed: Boolean = compareTo(MarathonTaskStatus.Failed)
+    def isFinished: Boolean = compareTo(MarathonTaskStatus.Finished)
+    def isKilled: Boolean = compareTo(MarathonTaskStatus.Killed)
+    def isKilling: Boolean = compareTo(MarathonTaskStatus.Killing)
+    def isLost: Boolean = compareTo(MarathonTaskStatus.Lost)
+    def isMesosLost: Boolean = isLost || isGone || isUnreachable
```

Comment:
We shouldn't need or have this. the _old_ Mesos `LOST` doesn't tell much about what's actually going on with that task, so I'd rather explicitly handle the new, better states (even if they're translated for now)